### ISSUE TITLE
Help needed on how to scrape full length movie files.

### ISSUE THREAD

**Issue by delegreg** [DirtyRacer1337/Jellyfin.Plugin.PhoenixAdult#50]
I'm trying to scrape a big collection of full length movies, tried setting the default provider to adult dvd empire but most searches come back empty (I can't seem to find the results from the scraper)

Running on jellyfin docker image, using the repository manifest to install, configuration is 'AdultDVDEmpire' as default site.

Log extract follows : 
```
[2021-03-28 12:56:19.445 +02:00] [INF] [95] PhoenixAdult.Provider: searchInfo.Name: Batman XXX A Porn Parody
[2021-03-28 12:56:19.446 +02:00] [INF] [95] PhoenixAdult.Provider: newTitle: AdultDvdEmpire  Batman XXX A Porn Parody
[2021-03-28 12:56:19.447 +02:00] [INF] [95] PhoenixAdult.Provider: site: 45:0 (AdultDVDEmpire)
[2021-03-28 12:56:19.447 +02:00] [INF] [95] PhoenixAdult.Provider: searchTitle: Batman XXX A Porn Parody
```

**Comment by DirtyRacer1337** [comment]
`AdultDvdEmpire` not available on [2.5.1.43](https://github.com/DirtyRacer1337/Jellyfin.Plugin.PhoenixAdult/tree/2.5.1.43), you can use [Nightly Build](https://github.com/DirtyRacer1337/Jellyfin.Plugin.PhoenixAdult/releases/tag/nightly)

**Comment by delegreg** [comment]
Got it running, thanks for the help.
